packages/astro/scripts/build.js

```diff
@@ -34,3 +35,6 @@ if (existsSync('./dist/default')) {
 
 // copy default folder unmodified
 await cp('./src/default', './dist/default', { recursive: true });
+
+// remove tests files
```

Comment:
```suggestion
// remove test files
```

Reply:
Also, can't we just exclude the `.spec.ts` files from being emitted to the `dist` folder?

Reply:
The default folder is copied verbatim on line 37. This is why we have to remove those files after copying them.